pkg/db/db.go

```diff
@@ -638,6 +783,7 @@ func (h *DBHandler) RunCustomMigrations(
 	ctx context.Context,
 	getAllAppsFun GetAllAppsFun,
 	getAllDeploymentsFun GetAllDeploymentsFun,
+	getAllReleasesFun GetAllReleasesFun,
```

Comment:
Some of these functions read from the manifest repo and some read from the database. Let's put an explanatory comment for this as it's a bit unintuitive (and perhaps something we need to fix in the future).